wagtail/contrib/search_promotions/models.py

```diff
@@ -50,12 +69,28 @@ def garbage_collect(cls):
 
     @classmethod
     def get(cls, query_string):
+        """
+        Gets or creates object matching query_string.
+
+        Args:
+           - query_string(str): Query text.
+
+        Returns:
+           - First Query object matching query_string.
+        """
         return cls.objects.get_or_create(
             query_string=normalise_query_string(query_string)
         )[0]
 
     @classmethod
     def get_most_popular(cls, date_since=None):
+        """
+        Get most popular promotion, optionally since a date.
+
+        Args
+            - date_since (datetime.date, optional, but not yet implemented). Earliest date. Default is all.
+        """
+
```

Comment:
```suggestion
```

For now, remove this, feel free to put into your other PR which includes the usage of `date_since`.